tide-log/src/lib.rs

```diff
@@ -1,5 +1,7 @@
+#![cfg_attr(feature = "nightly", deny(missing_docs))]
```

Comment:
I noticed that none of the `Cargo.toml` actually have a `nightly` feature in them. I think this being written like this in other `lib.rs` is just an artifact from an older setup, and it should probably move into the general `warn` section.

Reply:
Removed this. However, adding this on to warn opens up a lot of warning for all the middleware crates. Middleware structs are documented, however not the crates. Suggestions?

Reply:
@Nemo157 - would you like me to add `missing_docs` to warn, or leave it out for the moment? (Adding it will fail all crates except for `tide` currently)

Reply:
They should probably have some documentation added, maybe open an issue about filling out the documentation and adding it to each of the crates.